install/example/freebsd_freenas.md

```diff
@@ -187,28 +166,53 @@ Starting tomcat85.
 
 #### Navigate to Airsonic
 
-In a browser. Take your `SERVER_IP` and `PORT` and found in [Test Tomcat](#test-tomcat), 
-and navigate there in a web browser
-
-> In this example install, we would browse to:
-> ```http://10.0.0.10:8080```
+Browse to http://IPADRESS:8080
 
 #### Log into Airsonic
 
 Log in. The default is username: admin password: admin
 
 Follow the prompts on the web page to change the password. This will log you out. Please re-login with your new password
 
-#### Configure media directories in system or jail
+#### Mount media directories in jail (FreeNAS)
 
-If you are using FreeNAS, you will mount your media files using the FreeNAS gui.
+First be aware that you have enabled group access to your media pool dataset
+In FreeNAS gui goto Storage>Pools, choose your media pool and choose "Edit ACL".
 
+Then Stop the jail and mount your media folder to the jail (Jails>Mount Points)
+You should mount to ```/root/var/music``` (music folder doesn't exist so you need to manually type in music to create this folder). Then save and start the jail again.
+
+In jail shell:
+```
+# ls -l /var
+drwxrwx---+ 321 nobody   1000     322 Apr  4 20:32 music
+```
+Notice that the music folder is accessible by group 1000 in this case, so we need to add the user "www" to this group to have group access to the folder. The groupID 1000 is actually from outside the jail, so we first need to make a corresponding group inside the jail (you need to change 1000 to the correct groupID in your setup).
```

Comment:
I don't run FreeBSD, but what if the group 1000 already exists (from where I stand it's a fair assumption, it's the default user group in many Linux distributions)?